docs/user/parser.rst

```diff
@@ -40,6 +69,33 @@ Visual Parsers
 
 The following docs describe various visual parsers. They parse visual information,
 e.g., bounding boxes of each word.
+Fonduer can parse visual information only for hOCR and HTML files
+with help of :class:`HocrVisualParser` and :class:`PdfVisualParser`, respectively.
+It is recommended to provide documents in hOCR instead of HTML,
+because :class:`PdfVisualParser` is not always accurate by its nature and assigns
```

Comment:
Out of curiosity, how much better is hOCR than PDF?

Reply:
hOCR should be 100% accurate by its design (otherwise there is an error in implementation) because bboxes and words are completely tied in hOCR. Of course it is not always one-to-one mapping between spacy tokens and hOCR tokens. E.g., `"Apple."` in hOCR is tokenized (by spaCy) into `["Apple", "."]`. So you would have to expect this one-to-multi (and also multi-to-one) to be happening because spaCy and OCR engines tokenize slightly differently.

On the other hand, when you use HTML/PDF and `PdfVisualParser`, it is not the case.
The primary reason for this is that Fonduer can't control how PDF is converted to HTML (and vice verse) and doesn't even know how it is done.
As a result, Fonduer doesn't know the order of words in HTML and can only guess, and ended up fuzzy-matching words between HTML and PDF.